USER_MANUAL.md

```diff
@@ -82,19 +82,29 @@ The ReplicatedStartupActorTest is failing, pending https://improbableio.atlassia
   * Press "K" to delete a cube in the scene (used for debugging actors deleted while locked).
 
 
-##### Ability locking gym
+##### Ability activation gym
 * Demonstrates that:
+  * Gameplay abilities can be activated across server boundaries.
   * An Actor will be locked from crossing servers while a gameplay ability is running on it.
   * A player ownership hierarchy of Actors (controller, character & state) will not migrate while one Actor in the hierarchy is locked.
 * Contains:
   * a cube moving across a server boundary, with a gameplay ability granted to it which takes 4 seconds to complete.
   * a player with a gameplay ability granted to it which takes 4 seconds to complete.
 * To test the gym:
   * If it is working correctly the authority and authority intent of the cube can be seen to change as it moves across the floor, and the text "Uninitialized" hovering over the cube.
-  * Press "Q" to start the ability on the player.
-  * Press "T" to start the ability on the cube.
+  * Activate an ability on the player or the cube:
+    * Press "Q" to activate the ability on the player.
```

Comment:
Could you please give a quick description of what the user should expect to see if these key presses succeed/fail

Reply:
The text below the list of inputs describes the results of activating an ability - the intent here is that all of the different key presses activate the same ability and thus have the same visual result.
I've clarified the description of what success looks like, lmk if you think this still needs to be made clearer.